make/locations.mk

```diff
@@ -137,16 +134,21 @@ PODMAN_RUN_TEST := ${PODMAN_RUN} ${DOCKER_INTERACTIVE} --init
 DOCKER_HOME_PATH := /home/appuser
 DOCKER_KM_TOP := ${DOCKER_HOME_PATH}/km
 DOCKER_BLDTOP := ${DOCKER_KM_TOP}/build
+DOCKER_OPT_KONTAIN := ${DOCKER_BLDTOP}/opt/kontain
+DOCKER_OPT_KONTAIN_BIN := ${DOCKER_OPT_KONTAIN}/bin
+DOCKER_COVERAGE_KM := ${DOCKER_OPT_KONTAIN}/coverage/km
+DOCKER_OPT_COVERAGE := ${DOCKER_OPT_KONTAIN}/coverage
+DOCKER_OPT_COVERAGE_BIN := ${DOCKER_OPT_KONTAIN}/coverage/bin
 DOCKER_COVERAGE_KM_BLDDIR := ${DOCKER_BLDTOP}/km/coverage
 
-# These volumes needs to be mapped into runenv images. At the moment, they
-# contain km and km linkers.
-ifeq (${BLDTYPE}, ${COV_BLDTYPE})
+ifeq (coverage, $(filter coverage,$(MAKECMDGOALS)))
 	KM_OPT_BIN_PATH := ${KM_OPT_COVERAGE_BIN}
 else
 	KM_OPT_BIN_PATH := ${KM_OPT_BIN}
 endif
 
+# These volumes needs to be mapped into runenv images. At the moment, they
+# contain km and km linkers.
```

Comment:
Linkers?

Reply:
comment was there  before , again no idea why it is here